### PR TITLE
feat: support deleting service

### PR DESCRIPTION
**Pull request by zccabb**

### CODE REVIEW

File: cmd/resource/delete.go [hunk 1]
```diff
@@ -37,6 +38,16 @@ var (
...
 		},
+		"service": func(id sdk.ID) {
+			cluster, err := cloud.DefaultClient.GetDefaultCluster()
+			if err != nil {
+				output.Errorf("Failed to get the default cluster: %s", err.Error())
+			}
+			if err := cloud.DefaultClient.DeleteService(cluster.ID, id); err != nil {
+				output.Errorf("Failed to delete service: %s", err.Error())
+			}
+			fmt.Println("Delete service success , ID: ", id)
```
Comment (tokers) [inline cmd/resource/delete.go:49]: Don't need to print this line.

File: internal/cloud/api.go [hunk 2]
```diff
@@ -298,6 +298,18 @@ func (a *api) GetService(clusterID cloud.ID, appID cloud.ID) (*cloud.Application
...
 }
 
+func (a *api) DeleteService(clusterID cloud.ID, appID cloud.ID) error {
+	err := a.sdk.DeleteApplication(context.TODO(), appID, &cloud.ResourceDeleteOptions{
+		Cluster: &cloud.Cluster{
+			ID: clusterID,
+		},
+	})
+	if err != nil {
+		return errors.Wrap(err, "failed to delete service iterator")
```
Comment (tokers) [inline internal/cloud/api.go:308]:
```suggestion
		return errors.Wrap(err, "failed to delete service")
```

**Review by mscb402 — APPROVED** [review]

File: internal/cloud/api.go (hunk 2 above)
Comment (zccabb) [reply internal/cloud/api.go]: fixed

File: cmd/resource/delete.go (hunk 1 above)
Comment (zccabb) [reply cmd/resource/delete.go]: Done

**Review by Hazel6869 — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Pull request merged**